src/System.Threading.Tasks.Dataflow/tests/Dataflow/EtwTests.cs

```diff
@@ -9,6 +9,7 @@ namespace System.Threading.Tasks.Dataflow.Tests
     public class EtwTests
     {
         [Fact]
+        [ActiveIssue(3520, PlatformID.OSX)]
```

Comment:
I see.  This could be an interaction between my recent changes to speed up the test runs and how run-tests.sh builds out the overlay.  I think that perhaps what is happening is that previously we would never copy down the WP8 version and now we are and run-test.sh's way of building the overlay could be picking it up and using it instead of the regular version.  Thanks for the pointer.